content/en/docs/how-tos/adding-a-cluster-profile.md

```diff
@@ -92,6 +92,14 @@ changes.  These are:
   named `cluster-profile-<name>`, in which case a single new `switch` label is
   required.
 
+## Making a Cluster Profile Private
```

Comment:
I would suggest making this text a little bit formal:
```
Title: Private Cluster Profile
To restrict the usage of your cluster profile to specific organizations and repositories, you can create a pull request in the openshift/release repository. Within the pull request, add your repository or organization to the allowlist.

For detailed instructions, please refer to the README file.
```

Reply:
Thanks for suggestions, I reworded the text